app/views/planning_applications/conditions/_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= form_with model: [@planning_application],
+              local: true,
+              builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+              class: "govuk-!-margin-top-7",
+              url:,
```

Comment:
How about adding the data-controller here?

`html: { data: {controller: "add-condition"} }`